react/components/Animation/index.js

```diff
@@ -39,32 +19,41 @@ export default class Animation extends Component {
     className: PropTypes.string,
     /* Type of animation */
     type: PropTypes.oneOf(['drawerLeft', 'drawerRight', 'drawerTop', 'drawerBottom']),
+    /* The animation's duration in seconds */
+    duration: PropTypes.number,
+    /* The active animation deslocation in percentage */
+    transfer: PropTypes.number,
+    /* The not active animation deslocation in percentage */
+    transferEnter: PropTypes.number,
   }
 
   static defaultProps = {
     className: '',
     type: 'drawerLeft',
+    duration: ANIMATION_TIME,
+    transfer: ANIMATION_TRANSFER,
+    transferEnter: ANIMATION_TRANSFER_ENTER,
   }
 
-  renderChildren = style => (
-    <div className={this.props.className}
-      style={style}
-    >
-      {this.props.children}
-    </div>
-  )
+  getAnimation = () => {
```

Comment:
This is a function that computes a value based on props passed to the component, shouldn't be a _getter function_ then? Just like the functions defined in [this component](https://github.com/vtex-apps/search-result/blob/master/react/FilterNavigator.js).

Reply:
If you do that you could just pass `style={this.animation}` in line 54 of this file.